### DISCUSSION TITLE
Previewing generated config

### DISCUSSION THREAD

**Discussion by PiotrMachowski** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#321] [Q&A] [answered]
If you want to to preview generated config for current preset you have to double-click "play" button. It will be printed in JSON format in browser's console, to convert it to YAML you can use [json2yaml.com](https://www.json2yaml.com/).

**Comment by linceaerian** [comment]
Hello @PiotrMachowski,

A question about this, is this a method that can display the preset for the generated config ?
I'm currently looking for the presets of [Tasshack/dreame-vacuum](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/templates/tasshackDreameVacuum.md) integration in order to correct the tiles in errors and customize it:


![image](https://github.com/user-attachments/assets/e7ff207c-bcd2-4616-a4b7-61ce14d848cd)

But I do not find any doc that would display the current 'yaml' of the icons/tiles, etc, I only have the "generated" one (so I cannot copy paste the current tiles).

Also for the README about the custom tiles and icons, it could be great to include the main element (tides: or icons:) in the example, or in a line, cause I had to go to "discussions" to understand how to add them :)

**Reply by PiotrMachowski** [reply, marked as answer]
There is a button "Generate static config" that puts all the config into yaml for you to adjust it. Is it something that works for you?

**Reply by linceaerian** [reply]
Yes, perfect for me, exactly what I was looking for, I can copy the parts I want and stay in dynamic with this too.

**Reply by PiotrMachowski** [reply]
Great 👍
